scootapi/client.go

```diff
@@ -0,0 +1,101 @@
+package scootapi
```

Comment:
I haven't reviewed this but make sure to check out how workerapi/client presents both CLI and actual go client interfaces.